packages/api/endpoints/granules.js

```diff
@@ -440,35 +442,35 @@ async function del(req, res) {
   );
   log.info(`granules.del ${granuleId}`);
 
-  let dynamoGranule;
   let pgGranule;
-
+  let esResult;
   try {
+    // TODO - Phase 3 - we need a ticket to address granule/collection consistency
+    // For now use granule ID without collection search in ES
     pgGranule = await getUniqueGranuleByGranuleId(knex, granuleId);
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
-      if (!(await esGranulesClient.exists(granuleId))) {
+      // TODO - Phase 3 - we need to require the collectionID, not infer it
+
+      esResult = await esGranulesClient.get(granuleId);
+
+      if (esResult.detail === 'Record not found') {
         log.info('Granule does not exist in Elasticsearch and PostgreSQL');
         return res.boom.notFound('No record found');
       }
-      log.info(`Postgres Granule with ID ${granuleId} does not exist but exists in Elasticsearch. Proceeding with deletion.`);
+      if (esResult.detail === 'More than one record was found!') {
+        return res.boom.notFound('No Postgres record found, multiple ES entries found for deletion');
+      }
+      log.info(`Postgres Granule with ID ${granuleId} does not exist but exists in Elasticsearch.  Proceeding to remove from elasticsearch.`);
```

Comment:
deleteGranuleAndFiles has been updated to delete the ES record if pgGranule is not defined.